libmamba/src/solver/libsolv/database.cpp

```diff
@@ -30,17 +29,24 @@ namespace mamba::solver::libsolv
 {
     struct Database::DatabaseImpl
     {
-        explicit DatabaseImpl(specs::ChannelResolveParams p_channel_params)
-            : matcher(std::move(p_channel_params))
+        explicit DatabaseImpl(specs::ChannelResolveParams p_channel_params, Settings settings_)
+            : settings(std::move(settings_))
+            , matcher(std::move(p_channel_params))
         {
         }
 
+        Settings settings;
```

Comment:
I understand that the addition of `settings` is the change which causes the ABI break you have mentioned in the description of this PR. Can you confirm?